src/App.jsx

```diff
@@ -1,13 +1,196 @@
-
 import './App.css'
+import FruitCounter from "./components/FruitCounter/FruitCounter.jsx";
+import {useState} from "react";
+import Input from "./components/Input/Input.jsx";
+import Button from "./components/Button/Button.jsx";
 
 function App() {
+    const [counters, setCounters] = useState({
+        strawberries: 0,
+        bananas: 0,
+        apples: 0,
+        kiwis: 0
+    });
```

Comment:
Je had hier een variabele kunnen maken met een object voor al het fruit. Vervolgens had je deze variabele als initiële state kunnen laten fungeren + in de reset functie kunnen aanroepen:

Als volgt:

```
function App() {
const initialFruitState = {
        strawberries: 0,
        bananas: 0,
        apples: 0,
        kiwis: 0
    }

const [counters, setCounters] = useState(initialFruitState)

const resetCounters = () => {
setCounters(initialFruitState)
}
```

Op deze manier maak je dus maar 1x het object aan.